kuiper-game/Systems/CaptainsConsole.cs

```diff
@@ -8,16 +8,27 @@ namespace Kuiper.Systems
     public class CaptainsConsole : ICaptainsConsole
     {
         private readonly ICaptainService _captainService;
-        private readonly Captain _currentCaptain;
 
         public CaptainsConsole(ICaptainService captainService)
         {
             _captainService = captainService;
 
-            _currentCaptain = _captainService.SetupCaptain();
+            _captainService.SetupCaptain();
         }
         public void ConsoleMapper(string input)
         {
+            if(input.StartsWith("ship"))
```

Comment:
Trying this out to see if it's easier on the eyes, it's not super duper, but better than a switch from hell